tests/performance-test/performance-test.sh

```diff
@@ -49,91 +49,170 @@ ellipse(){
            ELLIPSE=".  "
     ;;
     esac
-}
-
-make_grafana_datasources(){
-    echo "Creating OCP datasource"
-    RESP=$(curl --silent --output /dev/null -d "$OCP_DATASOURCE" -H 'Content-Type: application/json' "$GRAF_HOST/api/datasources")
-    if echo "$RESP" | grep -q "RequiredError"; then
-        echo "Unable to create OCP datasource with reply: "
-        echo "$RESP"
-        exit 1
-    fi
-
-    echo "Creating SAF datasource"
-    RESP=$(curl --silent --output /dev/null -d "$SAF_DATASOURCE" -H 'Content-Type: application/json' "$GRAF_HOST/api/datasources")
-    if echo "$RESP" | grep -q "RequiredError"; then
-        echo "Unable to create SAF datasource with reply: "
-        echo "$RESP"
-        exit 1
-    fi
+    printf "\r"
 }
 
 make_service_monitors(){
     oc apply -f ./grafana/prom-servicemonitor.yml 
     oc apply -f ./grafana/qdr-servicemonitor.yml
 }
 
+# create edge router for more realistic env setup: telemetry-bench -> qdr -> qdr -> SG
 make_qdr_edge_router(){
     if ! oc get qdr qdr-test; then
-        echo "Deploying edge router"
-        oc create -f <(sed -e "s/<<REGISTRY_INFO>>/$(oc registry info)/" ./deploy/qdrouterd.yaml.template)
+        printf "\n*** Deploying Edge Router ***\n"
+        sed -e "s/<<REGISTRY_INFO>>/$(oc registry info)/g" ./deploy/qdrouterd.yaml > /tmp/qdr-test.yaml
```

Comment:
Don't like my one-liner? Pulled it apart to find the missing `g` and never put it back together? ;) Not offended.

Reply:
I think the second thing is what happened. I like the one liner!

Reply:
I guess you could technically put it back to the one-liner since you use that same method in other places throughout?

Reply:
Yes I definitely can